packages/wxt/src/core/create-server.ts

```diff
@@ -147,9 +147,6 @@ function createFileReloader(server: WxtDevServer) {
   return async (event: string, path: string) => {
     await wxt.reloadConfig();
```

Comment:
Last change - let's move this down to right before line 160 (before `detectDevChanges` is called) so it only runs once if multiple files are changed.

Reply:
Yeah, it's nice at call the queue execution timing. Commit pushed.